.gitignore

```diff
@@ -148,6 +148,10 @@ integration_tests/mock/target/
 integration_tests/mock/dbt_modules/
 integration_tests/mock/regular_output
 integration_tests/mock/fal_output/agent_wait_time_test
-integration_tests/mock/.user.yml
+integration_tests/projects/001_flow_run_with_selectors/.user.yml
+integration_tests/projects/**/logs/**
+integration_tests/**/logs/
+integration_tests/**/.user.yml
 integration_tests/mock/script_selected
 integration_tests/mock/false_script_selected
+integration_tests/mock/.user.yml
```

Comment:
Can we change this to:

```
.user.yml
```

This will ignore files with that name in all directories. We can do the same for

```
logs/
target/
dbt_modules/
```